### ISSUE TITLE
1.7.10

### ISSUE THREAD

**Issue by zxterProxy** [Daveyx0/PrimitiveMobs#302]
Is the source code still on 1.7.10? 
Could I get it ?

**Comment by Daveyx0** [comment]
I might be able to dig it up if I look hard. Any reason you want to get that old source?

**Comment by zxterProxy** [comment]
Yes I have legacy the server is on 1.7.10 and I support some old fashion and write a new one. A hobby, so to speak ) I would like to take some mobs from your dungeon mod. That would be really cool!

**Comment by zxterProxy** [comment]
I gathered a small team of enthusiasts and we chose the most atmospheric mods and based on minefantasy 2 and thaumcraft 4 we make a very large and well-developed build + lore and new mods ) We really liked your mod! We like this style of semi minecraft XD
We still draw our own texture pack )

**Comment by zxterProxy** [comment]
You couldn't find the source code ?

**Comment by Omgise** [comment]
@zxterProxy Maybe this link could help you: https://github.com/Rozmir-Rohi/primitive-mobs-1.7.10-rozmirs-fork
